cli/dcoscli/package/main.py

```diff
@@ -308,8 +314,16 @@ def _install(package_name, options_path, app_id, cli, app, yes):
               "repositories"
         raise DCOSException(msg)
 
-    # TODO(CD): Make package version to install configurable
-    pkg_version = pkg.latest_version()
+    version_dir = [k for k, v in iteritems(pkg.software_versions())
```

Comment:
What's your point with that screenshot?  `software_versions()` returns a dict whose values are package versions, so based on that, it seems that `pkg_version` is a package version, not a software version